static/app/utils/findClosestNumber.tsx

```diff
@@ -1,4 +1,6 @@
-// Finds the closest match to the number from the provided numbers array
+/**
+ * Finds the closest match to the number from the provided numbers array
```

Comment:
I thought we only use `docBlock` for describing  props of a component

Reply:
nah, it's super useful for any exported anything! since it shows up in the hover tooltip things
<img width="638" alt="image" src="https://user-images.githubusercontent.com/1421724/191911109-4ca4f73a-77ee-451a-98c2-717884dbf751.png">